### ISSUE TITLE
New temporary branch ds-bingchu-fork-1 for slow sim maker integration

### ISSUE THREAD

**Issue by plexoos** [star-bnl/star-ist#4]
@bingchuhuang 
I tried to split into smaller commits https://github.com/star-bnl/star-ist/tree/ds-bingchu-fork-1

There is still one huge commit (https://github.com/star-bnl/star-ist/commit/c99335a16173c0fdf0baa8a452e24b8b3149107c) that would be nice to split into smaller ones. It is hard to follow the logic of all changes as it is right now

**Comment by bingchuhuang** [comment]
I will write some slides about the changes, it is not easy to follow the changes from diff two versions. The hit loop changes from per APV to per Ladder as we have to load all raw hits from data and simu before we select them. I will send the slides to you soon.

**Comment by plexoos** [comment]
not relevant anymore
